lte/gateway/python/magma/subscriberdb/protocols/s6a_proxy_servicer.py

```diff
@@ -125,9 +125,8 @@ def UpdateLocation(self, request, context):
 
             sec_apn.ambr.max_bandwidth_ul = apn.ambr.max_bandwidth_ul
             sec_apn.ambr.max_bandwidth_dl = apn.ambr.max_bandwidth_dl
-            sec_apn.pdn = (
-                s6a_proxy_pb2.UpdateLocationAnswer.APNConfiguration.IPV4
-            )
+            sec_apn.pdn = apn.pdn
+
```

Comment:
This will break the operations with orc8r as there is no pdn type configuration exposed by Orc8r API yet. First check if apn.pdn is defined otherwise assign `s6a_proxy_pb2.UpdateLocationAnswer.APNConfiguration.IPV4` as the default.